<?xml version="1.0"?>
<launch>
    <!--Transform Publisher-->
    <node pkg="sensor_fusion" type="transform_broadcast" name="transform_broadcast">
        <param name="target_frame" type="string" value="map" />
        <param name="source_frame" type="string" value="centerlaser" />
    </node>
</launch>
